src/common/components/testimonial/testimonial.css

```diff
@@ -1,11 +1,9 @@
-.pullQuote > p:first-of-type:before,
-.pullQuote>p:last-of-type:after {
-  color:#cdcdcd;
-  display:inline-block;
-  font-size:1em;
-  font-weight:700;
-  line-height:1rem;
-  max-width:1.25rem
+.pullQuote > p:first-of-type::before,
+.pullQuote > p:last-of-type::after {
+  color:#dd4814;
```

Comment:
I thought the Ubuntu orange had changed, but I might have dreamt that ... anyway, should this be a variable?